docs/track.rst

```diff
@@ -122,6 +123,10 @@ In the ``operations`` section you describe which operations are available for th
 
 In the ``schedule`` section you describe the workload for the benchmark, for example index with two clients at maximum throughput while searching with another two clients with ten operations per second. The schedule either uses the operations defined in the ``operations`` block or defines the operations to execute inline.
 
+In the ``challenges`` section you describe more than one set of operations, in the event your track needs to test more than one set of scenarios. This section is optional, and more information on ``challenges`` :ref:`can be found here <track_challenge>`.
+
+Creating a track does not require all of the above sections to be used. Tracks that are used against existing data may only rely on querying ``operations`` and can omit the ``indices``, ``templates``, and ``corpora`` sections.
```

Comment:
And ideally link to https://esrally.readthedocs.io/en/stable/track.html#a-track-with-a-single-task which shows a nice query example.